Providers/Scripts/2.6x-2.7x/Scripts/nxOMSAutomationWorker.py

```diff
@@ -31,7 +31,7 @@ def Set_Marshall(WorkspaceId, Enabled, AzureDnsAgentSvcZone):
 
 def set_marshall_helper(WorkspaceId, Enabled, AzureDnsAgentSvcZone, mock_worker_config_file=False):
     try:
-        isprimary = is_primary_workspace(WorkspaceId)
+        isprimary = is_oms_primary_workspace(WorkspaceId)
```

Comment:
is_primary? Try be consistent in naming when possible, makes the code inconsistent otherwise..